src/assembly_checker_main.cpp

```diff
@@ -137,7 +158,8 @@ int main(int argc, char** argv)
         auto vcf_path = vm[ebi::vcf::INPUT].as<std::string>();
         auto fasta_path = vm[ebi::vcf::FASTA].as<std::string>();
         auto fasta_index_path = fasta_path + ".fai";
-        auto outputs = get_outputs(vm[ebi::vcf::REPORT].as<std::string>(), vcf_path);
+        auto outdir = get_output_path(vm[ebi::vcf::OUTDIR].as<std::string>(), vcf_path);
+        auto outputs = get_outputs(vm[ebi::vcf::REPORT].as<std::string>(), outdir);
```

Comment:
it would be nice to log which is the output path, just as the validator does:
[info] Summary report written to : ../empty.vcf.errors_summary.1536308696324.txt

we would need to do it for the text report and the valid report, so the valid PR needs to be after this PR